audit/audit-gcp.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -x -e
+CNCF_GCP_ORG=758905017065
+
+# gcloud organizations describe $CNCF_GCP_ORG 2>&1
+# ERROR: (gcloud.organizations.describe)
+# User [hh@ii.coop] does not have permission to access organization []
+
+format=json
+gcloud iam roles list --organization=$CNCF_GCP_ORG --format=$format \
+       > cncf-org.roles.$format
+gcloud organizations get-iam-policy $CNCF_GCP_ORG --format=$format \
+       > cncf-org.policy.$format
+gcloud projects list \
+       --filter "parent.id=$CNCF_GCP_ORG" \
+       --format "value(name, projectNumber)" \
+    | while read PROJECT NUM; do \
+    export CLOUDSDK_CORE_PROJECT=$PROJECT
+    gcloud projects get-iam-policy $PROJECT --format=$format > $PROJECT.policy.$format
+    gcloud iam roles list --project $PROJECT --format=$format > $PROJECT.roles.$format
+    mkdir -p $PROJECT.roles
+    for ROLE_PATH in `gcloud iam roles list --project $PROJECT --format="value(name)"`
+    do
+        ROLE=`basename $ROLE_PATH`
+        gcloud iam roles --project=$PROJECT describe $ROLE \
+               --format=json > $PROJECT.roles/$ROLE.json
+    done
+    gcloud services list --filter state:ENABLED --format=$format > $PROJECT.services.$format
+    for service in `gcloud services list --filter state:ENABLED --format=json | jq -r .[].config.name`
+    do
+        case $service in
+            compute.googleapis.com)
+                echo TODO: Needs compute.projects.get
```

Comment:
'compute.projects.get' will also be necessary for 'gcloud compute project-info describe'

Reply:
Added compute viewer